runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -1645,13 +1644,25 @@ J9::ARM64::TreeEvaluator::monexitEvaluator(TR::Node *node, TR::CodeGenerator *cg
       generateMemSrc1Instruction(cg, fej9->generateCompressedLockWord() ? TR::InstOpCode::strimmw : TR::InstOpCode::strimmx,
                                  node, TR::MemoryReference::createWithDisplacement(cg, addrReg, 0), dataReg);
 
+#if JAVA_SPEC_VERSION >= 19
+      /**
+       * Adjust J9VMThread ownedMonitorCount for execution paths that do not
+       * go out of line to release the lock.  It is safe and efficient to do
+       * this unconditionally.  There is no need to check for underflow.
+       */
+      generateTrg1MemInstruction(cg, TR::InstOpCode::ldrimmx, node, tempReg, TR::MemoryReference::createWithDisplacement(cg, metaReg, fej9->thisThreadGetOwnedMonitorCountOffset()));
+      generateTrg1Src1ImmInstruction(cg, TR::InstOpCode::subimmx, node, tempReg, tempReg, 1);
+      generateMemSrc1Instruction(cg, TR::InstOpCode::strimmx, node, TR::MemoryReference::createWithDisplacement(cg, metaReg, fej9->thisThreadGetOwnedMonitorCountOffset()), tempReg);
+#endif
```

Comment:
How about adding these instructions in the comment at https://github.com/eclipse-openj9/openj9/blob/2184d058bc57a01b9736dff221810dbaa5637cac/runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp#L1617-L1618 ?